benchmark/single-source/UTF16Decode.swift

```diff
@@ -221,3 +226,11 @@ public func run_UTF16Decode_InitFromCustom_noncontiguous_ascii(_ N: Int) {
     blackHole(String(decoding: asciiCustomNoncontiguous, as: UTF16.self))
   }
 }
+
+@inline(never)
+public func run_UTF16DecodeLength(_ N: Int) {
+  for _ in 0..<10*N {
+    // Round trip time to go utf8 -> utf16 -> utf8 -> utf16 length
+    blackHole(String(decoding: japanese.utf16, as: UTF16.self).utf16.count)
```

Comment:
This won't have a ton of signal until the UTF16 decoding vectorization patch lands, but a quick local benchmark suggests breadcrumb formation is around 15% of the time even before then…

…except this string is too short, I'll fix that